db/migrate/20231222190024_add_email_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToUsers < ActiveRecord::Migration[7.1]
+  def change
+   #add_column :users, :email, :string
+  end
+end
```

Comment:
- The same issue is applicable here as you've added an empty migration file. Please, reevaluate to remove it.

Reply:
Dear @zieeco , 
Thanks for the suggestion, 
I already modified the change you suggested, 
Best wishes,